test/framework/src/main/java/org/elasticsearch/index/shard/IndexShardTestCase.java

```diff
@@ -356,7 +364,7 @@ protected IndexShard reinitShard(IndexShard current, ShardRouting routing, Index
                 routing,
                 current.shardPath(),
                 current.indexSettings().getIndexMetaData(),
-                null,
+                null, null,
```

Comment:
Nit: This is probably better on a separate line like the rest of the parameters.